diskquota_utility.c

```diff
@@ -1149,9 +1150,9 @@ get_rel_oid_list(void)
 
 	initStringInfo(&buf);
 	appendStringInfo(&buf,
-			"select oid "
-			" from pg_class"
-			" where oid >= %u and (relkind='r' or relkind='m')",
+			"select oid, relkind "
+			"from pg_class "
+			"where oid >= %u and (relkind='r' or relkind='m' or (relinkind = 'i'))",
```

Comment:
Nit: Keep coding style cosistent.
```suggestion
			"where oid >= %u and (relkind='r' or relkind='m' or relinkind = 'i')",
```
OR
```suggestion
			"where oid >= %u and ((relkind='r') or (relkind='m') or (relinkind = 'i'))",
```